opm/material/densead/DynamicEvaluation.hpp

```diff
@@ -518,22 +518,22 @@ class Evaluation<ValueT, DynamicSize>
 
 private:
 
-    FastSmallVector<ValueT, 9> data_;
+    FastSmallVector<ValueT, staticSize> data_;
 };
 
-template <class Scalar>
-using DynamicEvaluation = Evaluation<Scalar, DynamicSize>;
+template <class Scalar, unsigned staticSize = 9>
+using DynamicEvaluation = Evaluation<Scalar, DynamicSize, staticSize>;
 
 } // namespace DenseAd
 
-template <class Scalar>
-Opm::DenseAd::Evaluation<Scalar, -1> constant(int numDerivatives, const Scalar& value)
-{ return Opm::DenseAd::Evaluation<Scalar, -1>::createConstant(numDerivatives, value); }
+template <class Scalar, unsigned staticSize>
+Opm::DenseAd::Evaluation<Scalar, -1, staticSize> constant(int numDerivatives, const Scalar& value)
+{ return Opm::DenseAd::Evaluation<Scalar, -1, staticSize>::createConstant(numDerivatives, value); }
 
-template <class Scalar>
-Opm::DenseAd::Evaluation<Scalar, -1> variable(int numDerivatives, const Scalar& value, unsigned idx)
-{ return Opm::DenseAd::Evaluation<Scalar, -1>::createVariable(numDerivatives, value, idx); }
+template <class Scalar, unsigned staticSize>
+Opm::DenseAd::Evaluation<Scalar, -1, staticSize> variable(int numDerivatives, const Scalar& value, unsigned idx)
+{ return Opm::DenseAd::Evaluation<Scalar, -1, staticSize>::createVariable(numDerivatives, value, idx); }
 
 } // namespace Opm
 
-#endif // OPM_DENSEAD_EVALUATION-1_HPP
```

Comment:
typo, (does not matter). Slightly more relevantly, the code generator is currently slightly wrong: it should produce `OPM_DENSEAD_EALUATION1_HPP` instead of `OPM_DENSEAD_EALUATION-1_HPP`.